datastore/importers/nmdb-import-ipconfig/Parser.cpp

```diff
@@ -136,8 +151,13 @@ void
 Parser::addIface(const std::string& _name, const std::string& _type)
 {
   nmdo::Interface iface;
+  std::string whitespace = "\t\n\v\f\r ";
+  std::string type = _type;
+  type = type.erase(type.find_last_not_of(whitespace) + 1);
+  type = type.erase(0, type.find_first_not_of(whitespace));
+  
   iface.setName(_name);
-  iface.setMediaType(_type);
+  iface.setMediaType(type);
```

Comment:
This works, but I want to ensure you are aware (since I am assuming you are new to Boost Spirit):
If you changed line 118 to `+(qi::ascii::print - (+qi::blank >> "adapter"))` it would pass the interface type without the trailing space(s).  You should never get a space before as `ifaceTypeName`, which calls `ifaceType`, uses a `qi::ascii:blank_type` skipper.